### PR TITLE
Automatically Updating LTS Versions

### PR DESCRIPTION
**Pull request by brianluisgomez**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: .github/workflows/lts-release.yml
Comment (brianluisgomez) [reply .github/workflows/lts-release.yml:93]: It all depends on how much time we want to dedicate to something like that. It is definitely out of the scope of this card, though. We not only need an action like this but technically we need another one too that will automatically approve the PR since we need two approvals per our branch controls. So it may be the case that for the moment the better option is to just ignore the auto-merging part anyway until we get the approval side figured out (to automate we would need two other tokens so those different 'users' can do the approval)

**Review by ajrice6713 — APPROVED** [review]

**Review by ckoegel — APPROVED** [review]

**Pull request merged**
